src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseRepository.java

```diff
@@ -53,7 +53,8 @@ public CCDCaseRepository(
 
     @Override
     public List<Claim> getBySubmitterId(String submitterId, String authorisation, Integer pageNumber) {
-        return ccdCaseApi.getBySubmitterId(submitterId, authorisation, pageNumber);
+        List<Claim> result = ccdCaseApi.getBySubmitterId(submitterId, authorisation, pageNumber);
+        return result;
```

Comment:
I feel like it was better the way before as youre not using result (except for debugging and forgot?)

Reply:
that wasn't supposed to go in, it was a change to help debugging, I am reverting it, thanks